extensions/modules/persistentlogin/src/main/java/org/exist/xquery/modules/persistentlogin/PersistentLogin.java

```diff
@@ -60,9 +61,9 @@ public static PersistentLogin getInstance() {
 
     public final static int DEFAULT_TOKEN_LENGTH = 16;
 
-    public final static int INVALIDATION_TIMEOUT = 20000;
+    public final static int INVALIDATION_TIMEOUT = 1000;
```

Comment:
We need to check if this is safe to lower the timeout to 1 second, was 20 seconds.

Reply:
code quality issue (minor)